ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -93,6 +93,8 @@ open class NavigationBar: UINavigationBar {
         }
     }
 
+    @objc public static let navigationBarBackgroundColor = UIColor(dynamicColor: FluentTheme.shared.aliasTokens.colors[.background3])
```

Comment:
in mainbranch, we have this defined in FluentUIFramework.swift. is it purposefully moved to this location?

Reply:
Yes, it seemed to me that it would be more coherent for the nav bar background color to live inside the NavigationBar. Would you rather have it moved to FluentUIFramework?

Reply:
just making sure you are making intentional changes. it is also useful to put previous commit hash number to track the history of changes. the only little odd thing now is that navigationbar colors are technically defined in 2 different places. 
        func backgroundColor(fluentTheme: FluentTheme) -> UIColor {
            switch self {
            case .normal:
                return UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.background3])

Reply:
It's rewritten to return `Style.system.backgroundColor(fluentTheme: fluentTheme)`

Reply:
Should you update the pr description?

Reply:
should fluentframworke.swift refere to this then?

Reply:
FluentUIFramework.swift has already had navigationbarbackgroundcolor fully removed. Do you want to fully undo that change and replace all references with the new constant added here?